src/cmd/connectioncheck/connection_check.go

```diff
@@ -51,6 +51,7 @@ func sendTestMessage(endpoint, apiToken, clusterUid string, insecure bool) {
 	record.SetTimestamp(time.Now())
 
 	logger.Emit(ctx, record)
+	log.Print("Connection check was successful")
```

Comment:
Added so that the init container shows some success message instead of just ending. However, this change will actually be applied only after we release a new image.